python/cudf/bindings/cudf_cpp.pxd

```diff
@@ -629,4 +629,9 @@ cdef extern from "cudf.h" nogil:
     cdef gdf_error gdf_to_dlpack(DLManagedTensor *tensor,
                                  const gdf_column ** columns,
                                  gdf_size_type num_columns) except +
-    
\ No newline at end of file
+
+    cdef gdf_error gdf_nvtx_range_push(const char * const name, gdf_color color )
+
+    cdef gdf_error gdf_nvtx_range_push_hex(const char * const name, unsigned int color )
+
+    cdef gdf_error gdf_nvtx_range_pop()
```

Comment:
Should we add `except +` after these calls for future proofing against raising exceptions?

Reply:
@jrhemstad You may be the right person to answer whether we'd ever expect the `gdf_nvtx_*` functions to raise exceptions.

Reply:
Yeah, should just assume that any libcudf function can throw exceptions.